### PR TITLE
[13.0][MIG] analytic_tag_dimension_sale_warning

### PR DESCRIPTION
**Pull request by i-vyshnevska**
migrate analytic_tag_dimension_sale_warning
based on #280
build requires merge #264

### CODE REVIEW

File: analytic_tag_dimension_sale_warning/readme/HISTORY.rst [hunk 1]
```diff
@@ -0,0 +1,14 @@
+12.0.1.0.0 (2020-01-09)
```
Comment (jcoux) [inline analytic_tag_dimension_sale_warning/readme/HISTORY.rst:6]:
```suggestion
13.0.1.0.0 (2020-03-30)
~~~~~~~~~~~~~~~~~~~~~~~

Migrated to odoo 13.

12.0.1.0.0 (2020-01-09)
```

**Review by jcoux — APPROVED** [review]
Little suggestion, but not blocking.
Code review only.

**Review by TDu — APPROVED** [review]

**Comment by captivea-dcr** [comment]
@i-vyshnevska new based-on PR is #307 
Do you plan to work on this again?

**Comment by captivea-dcr** [comment]
@i-vyshnevska #310 has been merged. Do you plan to work on this one those days?

**Comment by kittiu** [comment]
Let's close this as it has replacement.

**Pull request closed without merging**